daily-challenges/2024-12/19/problem.md

```
@@ -0,0 +1,67 @@
+  mau: string;       // Mã màu
+}
+
+interface TrangThaiGame {
+  bangDiem: number;
+  capDo: number;
+  bangChoi: number[][];  // 0: ô trống, 1-5: các màu khối
+}
+```
+
+- Class `TetrisGame` với các phương thức:
+  + `diChuyenKhoi(huong: 'trai' | 'phai' | 'xuong')`: Di chuyển khối
```

Markdownlint (0.37.0)

Expected: dash; Actual: plus
Unordered list style

(MD004, ul-style)